examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -841,12 +841,12 @@ server cluster PowerSource = 47 {
     kReplaceable = 0x8;
   }
 
-  readonly attribute enum8 status = 0;
+  readonly attribute PowerSourceStatus status = 0;
   readonly attribute int8u order = 1;
   readonly attribute char_string<60> description = 2;
-  readonly attribute enum8 batteryChargeLevel = 14;
-  readonly attribute boolean batteryReplacementNeeded = 15;
-  readonly attribute enum8 batteryReplaceability = 16;
+  readonly attribute BatChargeLevel batChargeLevel = 14;
```

Comment:
This might be a problem on the autogen via zap

Reply:
I think it's not a bug, it's a feature :) Looks like @andreilitvin deliberately chose to use `attribute {{asLowerCamelCase name}}` int `MatterIDL.zapt` template.

Reply:
Lets not change, otherwise having both types and variable names having the same name would definately be some form of compile error at some point.

A naming convention of lower camel case for naming members and upper camel case for types makes total sense to me for any programming language.